### PR TITLE
Create target directory (/etc/ld.so.conf.d) if it doesn't exist 

### PR DESCRIPTION
**Pull request by ops**

### CODE REVIEW

File: LINUX/Makefile [hunk 1]
```diff
@@ -164,6 +164,9 @@ uninstall: $(call CREATE_TARGET,$(SUBDIRS_ALL_NON_OPTIONAL) $(SUBDIRS_OPTIONAL),
 	ldconfig || /bin/true
 
 ldconfig:
+	if [ ! -d "${DESTDIR}/etc/ld.so.conf.d" ]; then \
+	   mkdir "${DESTDIR}/etc/ld.so.conf.d"; \
```
Comment (spiro-trikaliotis) [inline LINUX/Makefile:168]: Wouldn't a "mkdir -p" be better, just in case that $(DESTDIR)/etc does not exist (yet)?

Comment (ops) [reply LINUX/Makefile]: Yes, indeed.

**Comment by spiro-trikaliotis** [comment]
Thank you! :)

**Comment by spiro-trikaliotis** [comment]
I am sorry that I made you do it and then I modified to use install instead.
While testing your changes, I found another issue and there I was remembered of the command install.

**Comment by ops** [comment]
Don't worry. Your solution is much better.

**Pull request merged**